src/renderer/shared/core/types/chain.ts

```diff
@@ -35,5 +35,5 @@ type ExternalValue = {
   url: string;
 };
 
-type ExternalType = 'history' | 'staking' | 'crowdloans';
+type ExternalType = 'history' | 'staking' | 'crowdloans' | 'proxy';
```

Comment:
Let's convert `ExternalType` to enum as well within `ChainOptions` task I've mention above